feature/quicksettings/src/main/java/com/google/jetpackcamera/feature/quicksettings/QuickSettingsScreen.kt

```diff
@@ -76,44 +73,43 @@ fun QuickSettingsScreen(
         animationSpec = tween()
     )
 
-    Column(
-        modifier = modifier
-            .fillMaxSize()
-            .background(color = backgroundColor.value),
-        verticalArrangement = Arrangement.Top,
-        horizontalAlignment = Alignment.CenterHorizontally
-    ) {
-        DropDownIcon(toggleIsOpen, isOpen = isOpen)
-
-        if (isOpen) {
-            Column(
-                modifier = Modifier
-                    .fillMaxSize()
-                    .alpha(alpha = contentAlpha.value)
-                    .clickable {
-                        // if a setting is expanded, close it. if no other settings are expanded, then close out of the popup
-                        if (shouldShowQuickSetting == IsExpandedQuickSetting.NONE) toggleIsOpen() else shouldShowQuickSetting =
-                            IsExpandedQuickSetting.NONE
-                    },
-                verticalArrangement = Arrangement.Center,
-                horizontalAlignment = Alignment.CenterHorizontally
-            ) {
-                ExpandedQuickSettingsUi(
-                    currentCameraSettings = currentCameraSettings,
-                    shouldShowQuickSetting = shouldShowQuickSetting,
-                    setVisibleQuickSetting = { enum: IsExpandedQuickSetting ->
-                        shouldShowQuickSetting = enum
-                    },
-                    onLensFaceClick = onLensFaceClick,
-                    onFlashModeClick = onFlashModeClick,
-                    onAspectRatioClick = onAspectRatioClick,
-                    //onTimerClick = onTimerClick,
-                )
-            }
-        } else {
-            shouldShowQuickSetting = IsExpandedQuickSetting.NONE
+    if (isOpen) {
+        Column(
+            modifier = Modifier
+                .fillMaxSize()
+                .background(color = backgroundColor.value)
+                .alpha(alpha = contentAlpha.value)
+                .clickable {
+                    // if a setting is expanded, click on the background to close it.
+                    // if no other settings are expanded, then close the popup
+                    if (shouldShowQuickSetting == IsExpandedQuickSetting.NONE)
+                        toggleIsOpen()
+                    else
+                        shouldShowQuickSetting = IsExpandedQuickSetting.NONE
```

Comment:
I was using if,else because this part only needs to know if a quicksetting is expanded or not, that way I wouldnt have to keep adding the enum names to essentially the "else" option.

Also I don't think I understand what you mean by using the enum for 2 different things, at least the way currently written.
The way the `shouldShowQuickSetting` works is that when a quick setting we have designated as expandable has an `onClick()` that will change the value of `shouldShowQuickSetting` to display the respective choices.
My intention is that clicking on the dark background should backstep level whatever is currently on the quick settings screen. if a setting is expanded, go back to the whole quick settings menu. if it isn't, then close out of the entire menu.

 I think I could rename the enum if it makes that a bit confusing

Reply:
Okay, I think I get it now. Would it be more appropriate to call it "currentExpandedQuickSetting"? Or something like that.

Don't have to address that in this PR